src/transition/script.c

```diff
@@ -998,8 +998,8 @@ script_compile(config_setting_t *setting, struct script_parse_config cfg, char *
 		struct script_context_info_internal *info, *next_info;
 		HASH_ITER(hh, context_table, info, next_info) {
 			HASH_DEL(context_table, info);
-			free(info);
 		}
+		free(info);
 	}
```

Comment:
this looks wrong, won't this leak `info`?

Reply:
I check again, but in its original position where the fault appeared.

Reply:
ok, but can you explain why this leaks? or are you making changes without understanding just to make warnings go away?